extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlCreate.java

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.extended.kubectl;
+
+import com.google.common.base.Strings;
+import io.kubernetes.client.Discovery;
+import io.kubernetes.client.common.KubernetesListObject;
+import io.kubernetes.client.common.KubernetesObject;
+import io.kubernetes.client.extended.kubectl.exception.KubectlException;
+import io.kubernetes.client.util.ModelMapper;
+import io.kubernetes.client.util.Namespaces;
+import io.kubernetes.client.util.generic.GenericKubernetesApi;
+import io.kubernetes.client.util.generic.KubernetesApiResponse;
+import io.kubernetes.client.util.generic.options.CreateOptions;
+import java.util.Optional;
+import java.util.Set;
+
+public class KubectlCreate extends Kubectl.ApiClientBuilder<KubectlCreate>
+    implements Kubectl.Executable<KubernetesObject> {
+
+  KubectlCreate() {}
```

Comment:
Delete since the default constructor is both no-arg and package private (afaik)

Reply:
>  package private (afaik)

https://stackoverflow.com/a/33633500/9568880 the default visibility of constructor aligns w/ the class, so we need to make it explicitly package-private